### ISSUE TITLE
[BUG] Can't paste login token into 'token' field

### ISSUE THREAD

**Issue by Karyudo** [SoMuchForSubtlety/f1viewer#232]
Followed instructions and used Firefox to copy token. Token will paste using Ctrl+V into Notepad, but won't paste into f1viewer.

**Comment by Myujinri** [comment]
Try right click and paste, it works for me

**Comment by Karyudo** [comment]
Yeah, tried that and no menu popped up. But I found that if I right-clicked in the header bar of the app, then I could choose paste from there! So I'm in.

Next problem is that I get nothing but a grey screen and no audio from any of the FP1 options....

**Comment by crater9893** [comment]
Its working for me , it takes a while to finish pasting as its a very long string (Arch Linux using konsole 21.12.3)

**Comment by AbeerBPi** [comment]
Same issue here, on windows. Can't copy paste using shortcuts or the right click method someone here suggested

**Comment by AbeerBPi** [comment]
> Same issue here, on windows. Can't copy paste using shortcuts or the right click method someone here suggested

Nvm it actually does work, you just right click on the header and paste is under "edit"

**Comment by IceTDrinker** [comment]
Win 10 here, it seems I literally cannot paste text in _any_ of the boxes, I guess it's a TUI thing?

**Comment by zak60u** [comment]
same issue here, ive tried ctrl + v and also by clicking on the app header but no luck with both

**Comment by CastorSmith89** [comment]
Same issue. Last race it worked but now it does not. Tested latest release and also build master git repo. does not work. Pasted in onenote++ and copied again. does not work.

**Comment by CastorSmith89** [comment]
I found a workaround. 
Open poweshell
to to your directory where your f1viewer.exe is present. (cd "<Paste your path>")
.\f1viewer.exe

The program starts and I was able to use cntrl+v to paste my token

**Comment by rohitk2607** [comment]
Still unable to paste the subscription key, any other workaround?

**Comment by mauricioquintela** [comment]
pasting with ctrl+shift+v should work, at least it did here

**Comment by LeProgGamer** [comment]
Hello, 

Same issue, i can't use my token :( how i can do ? Write everything ?
